### ISSUE TITLE
Default Hotkey (CTRL+R) clashes with reverse-i-search in BASH 

### ISSUE THREAD

**Issue by fprott** [MaartenBaert/ssr#752]
The default Hotkey for SSR is CTRL+R under Ubuntu.
The default Hotkey for reverse-i-search within the console is CTRL+R under Ubuntu.

If both are active SSR will siphon the command. This is not wanted since new users do not know/care what the hotkey is since this is a feature for power-users.
I use this software to capture experiments that I trigger via the console. I did not recognize that the hotkeys are the same. I just lost a lot of work and time.

Recommend to change to a less frequently used hotkey. I suggest SUPER + R.

**Comment by MaartenBaert** [comment]
Fair point, however it used to be Super+R but was changed to Ctrl+R for some reason that I don't remember. I personally prefer Super+R as well. I've changed it back.
